tests/data/usage_scenarios/healthcheck_using_start_interval.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Test Healthcheck
+author: David Kopp
+description: test
+
+services:
+  test-container-1:
+    image: alpine
+    depends_on:
+      test-container-2:
+        condition: service_healthy
+  test-container-2:
+    build:
+      context: ../slow-start-application
+    healthcheck:
+      test: pgrep 'tail'
+      start_interval: 1s
+      start_period: 5s
```

Comment:
In order to differentiate from `interval` here, which will also run to my understanding by default, you should force `interval` to a unreasonably high value of 1h or similar to disable it effectively

Reply:
done.